test/test_playbooks/tasks/repository.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: "Create katello repository"
+  vars:
+    - organization: "Test Organization"
+    - product_name: "product1"
+    - repository_name: "repo1"
+    - repository_content_type: "yum"
+    - repository_content_url: "https://repos.fedorapeople.org/pulp/pulp/demo_repos/zoo/"
+    - repository_download_policy: "on_demand"
+  katello_repository:
+    username: "{{ foreman_username }}"
+    password: "{{ foreman_password }}"
+    server_url: "{{ foreman_server_url }}"
+    verify_ssl: "{{ foreman_verify_ssl }}"
+    organization: "{{ organization_name }}"
+    name: "{{ repository_name }}"
+    product: "{{ product_name }}"
+    content_type: "{{ repository_content_type }}"
+    url: "{{ repository_url }}"
```

Comment:
I guess, this is `{{ repository_content_url }}`.

Reply:
I was going with `<module>_<param>` and so it would be `repository_url` just like `repository_content_type`